crates/lox-io/src/ndm/kvn/parser.rs

```diff
@@ -0,0 +1,1013 @@
+/*
+ * Copyright (c) 2023. Helge Eichhorn and the LOX contributors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+// KVN spec section 7.4 of https://public.ccsds.org/Pubs/502x0b3e1.pdf
```

Comment:
Can you add some additional context and turn this into a doc comment?

Reply:
It's not part of the user-facing doc. The user-facing doc is here: https://github.com/lox-space/lox/blob/5ddb3b2bb2b9881e64acca9f53b00b4d1df60441/crates/lox-io/src/ndm.rs

Reply:
It would be helpful for Lox devs too – I'm not exactly sure what the comment means 🙂

Reply:
Will expand the comment, but I am not sure about the doc comment part.

Reply:
Have a look.